### PR TITLE
chore(react-infobutton): Renaming content slot to info

### LINKED ISSUE #27236: InfoButton: Rename content prop to info and forward children to popover surface

### PR DESCRIPTION
**Pull request by sopranopillow**
## Previous Behavior

InfoButton's information slot's name is `content`.

## New Behavior

InfoButton's information slot's name is `info`.

## Related Issue(s)

- Related #27118
- Fixes #27236

### CODE REVIEW

File: change/@fluentui-react-infobutton-e5984e77-89ae-4c61-85e0-984121adb4e5.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "chore: Rename content slot to info slot.",
```
Comment (sopranopillow) [inline change/@fluentui-react-infobutton-e5984e77-89ae-4c61-85e0-984121adb4e5.json:3]:
```suggestion
  "comment": "chore: Rename content slot to info.",
```

File: packages/react-components/react-infobutton/src/components/InfoLabel/InfoLabel.types.ts [hunk 2]
```diff
@@ -30,9 +30,9 @@ export type InfoLabelSlots = {
  */
 export type InfoLabelProps = ComponentProps<Partial<InfoLabelSlots>, 'label'> & {
   /**
-   * The content of the InfoButton's popover.
+   * The information of the InfoButton's popover.
```
Comment (behowell) [inline packages/react-components/react-infobutton/src/components/InfoLabel/InfoLabel.types.ts:33]: FWIW, I think "content" still makes sense in this comment

**Review by behowell — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                      |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                  Change |
| :----------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr> | `130.601 kB`<br />`39.438 kB` | `130.577 kB`<br />`39.439 kB` |  `-24 B` <br />`1 B`  |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoLabel.fixture.js'>InfoLabel</abbr>   | `133.901 kB`<br />`40.489 kB` | `133.874 kB`<br />`40.488 kB` | `-27 B` <br />`-1 B`  |

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/34dec1d344736ea7be694d29c9429c40f5cb6c9e'>34dec1d344736ea7be694d29c9429c40f5cb6c9e</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-infobutton-e5984e77-89ae-4c61-85e0-984121adb4e5.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-infobutton-e5984e77-89ae-4c61-85e0-984121adb4e5.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27316&commitId=7f8708fb5c8b4d0af2c619481a6385807b48009a&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1899 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - Dark Mode.active.chromium.png|1108| Changed|
|InfoButton.default - High Contrast.active.chromium.png|1263| Changed|
|InfoButton.default.active.chromium.png|1771| Changed|
|InfoButton.size.rest.chromium.png|3374| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

**Review by micahgodbolt — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Pull request merged**